app/assets/src/components/views/samples/notifications.jsx

```diff
@@ -35,10 +36,10 @@ const publicSampleNotificationsByProject = samples => {
         localStorage.setItem(
           "dismissedPublicSamples",
           JSON.stringify(
-            new Set([
```

Comment:
there was a bug here b/c JSON.stringify of a set will always be "{}" https://stackoverflow.com/questions/31190885/json-stringify-a-set